antlr4/org/eclipse/jnosql/query/grammar/Query.g4

```diff
@@ -37,6 +37,7 @@ change: name '=' value;
 key: value;
 keys: value (','value)*;
 value: ( number | string | array | function | parameter | json);
+string_parameter: string | parameter;
```

Comment:
Maybe use "value_string" instead.

Reply:
I'll do it!

Reply:
Please, let me know when it is ready to merge.

Reply:
It's done! Let's wait for the check process to be completed!